src/main/java/com/rabbitmq/perf/MulticastParams.java

```diff
@@ -39,34 +41,40 @@
     private int minMsgSize = 0;
 
     private int timeLimit = 0;
-    private float producerRateLimit = 0;
-    private float consumerRateLimit = 0;
+    private float producerRateLimit = 0.0f;
+    private float consumerRateLimit = 0.0f;
     private int producerMsgCount = 0;
     private int consumerMsgCount = 0;
     private boolean consumerSlowStart = false;
 
     private String exchangeName = "direct";
     private String exchangeType = "direct";
-    private List<String> queueNames = new ArrayList<String>();
+    private List<String> queueNames = new ArrayList<>();
     private String routingKey = null;
     private boolean randomRoutingKey = false;
     private boolean skipBindingQueues = false;
 
-    private List<?> flags = new ArrayList<Object>();
+    private List<?> flags = new ArrayList<>();
 
     private int multiAckEvery = 0;
-    private boolean autoAck = true;
-    private boolean autoDelete = false;
+    private boolean autoAck = false;
+    private boolean autoDelete = true;
```

Comment:
I suppose those are new default values when relevant flags are not specified. The commit messages don't talk about this, so I'm just checking that this change was meant to be part of the branch.

Reply:
I made the defaults of `MulticastParams` consistent with the defaults of command line options in `PerfTest`, so the new tests (which are using directly `MulticastParams`) are as closed as possible to the command line invocation. This is breaking change, but we don't expect people to use `MulticastParams`, so I think this is acceptable in a feature release. I'll add an issue to reflect it in the changelog thought.

Reply:
Ok, perfect :-)